### PR TITLE
Bucket Slide out change

### PR DESCRIPTION
**Pull request by mwernimont**
Changed the minimize button to text that reads 'Hide Your Bucket'.  Kept it inside an i tag to keep the functionality attached to it.

### CODE REVIEW

File: coastal-hazards-portal/src/main/webapp/WEB-INF/jsp/components/front/slides/slider-bucket.jsp [hunk 1]
```diff
@@ -3,7 +3,7 @@
         <div id="application-slide-bucket-controlset"  class="application-slide-controlset row">
             <div class="col-md-12">
                 <div id="application-slide-bucket-controlset-control-close" class="pull-left">
-                    <i class="fa fa-minus-square-o" alt="minus inside a square"></i>
+                    <i id="hideYourBucket">Hide your Bucket</i>
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/webapp/WEB-INF/jsp/components/front/slides/slider-bucket.jsp]: Let's not camelCase here. In IDs and classes, let's go with dash separated words as we've been doing

File: coastal-hazards-portal/src/main/webapp/css/front/slider-bucket.css [hunk 2]
```diff
@@ -30,16 +30,19 @@
...
                 margin-left: -10px;
             }
             #application-slide-bucket-container > div > div#application-slide-bucket-controlset > div > div#application-slide-bucket-controlset-control-close:hover {
                 cursor: pointer;
             }
             #application-slide-bucket-container > div > div#application-slide-bucket-controlset > div > div#application-slide-bucket-controlset-control-close > i {
-                font-weight: bold;
                 color: #003366;
             }
+            #hideYourBucket{
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/webapp/css/front/slider-bucket.css]: camelCase -> dash-separated

**Comment by mwernimont** [comment]
That better?

**Comment by isuftin** [comment]
:+1:

**Pull request merged**